pkg/apis/pingcap.com/v1alpha1/types.go

```diff
@@ -90,9 +90,10 @@ type TidbClusterSpec struct {
 	TiKV            TiKVSpec            `json:"tikv,omitempty"`
 	TiKVPromGateway TiKVPromGatewaySpec `json:"tikvPromGateway,omitempty"`
 	// Services list non-headless services type used in TidbCluster
-	Services        []Service                            `json:"services,omitempty"`
-	PVReclaimPolicy corev1.PersistentVolumeReclaimPolicy `json:"pvReclaimPolicy,omitempty"`
-	Timezone        string                               `json:"timezone,omitempty"`
+	Services         []Service                            `json:"services,omitempty"`
+	PVReclaimPolicy  corev1.PersistentVolumeReclaimPolicy `json:"pvReclaimPolicy,omitempty"`
+	Timezone         string                               `json:"timezone,omitempty"`
+	MaxFailoverCount int32                                `json:"maxFailoverCount,omitempty"`
```

Comment:
Why add `MaxFailoverCount` in `TidbCluster`?

Reply:
Sorry, that's a typo when I fix the merge conflict in TiDBSpec

Reply:
Looks like this field is actually exists in master but is not used anywhere...

https://github.com/aylei/tidb-operator/blob/automated-cherry-pick-of-%23915-upstream-release-1.0/pkg/apis/pingcap.com/v1alpha1/types.go#L96